plugins/services/src/js/columns/ServicesTableActionsColumn.tsx

```diff
@@ -0,0 +1,201 @@
+import * as React from "react";
+import { Trans } from "@lingui/macro";
+import { Cell, Icon } from "@dcos/ui-kit";
+import { Dropdown, Tooltip } from "reactjs-components";
+import { Hooks } from "#SRC/js/plugin-bridge/PluginSDK";
+import { SystemIcons } from "@dcos/ui-kit/dist/packages/icons/dist/system-icons-enum";
+import { iconSizeXs } from "@dcos/ui-kit/dist/packages/design-tokens/build/js/designTokens";
+
+import { isSDKService } from "#SRC/js/utils/ServiceUtil";
+import Pod from "../structs/Pod";
+import Service from "../structs/Service";
+import ServiceActionLabels from "../constants/ServiceActionLabels";
+import * as ServiceStatus from "../constants/ServiceStatus";
+import ServiceTree from "../structs/ServiceTree";
+import { ServiceActionItem } from "../constants/ServiceActionItem";
+
+const DELETE = ServiceActionItem.DELETE;
+const EDIT = ServiceActionItem.EDIT;
+const MORE = ServiceActionItem.MORE;
+const OPEN = ServiceActionItem.OPEN;
+const RESTART = ServiceActionItem.RESTART;
+const RESUME = ServiceActionItem.RESUME;
+const SCALE = ServiceActionItem.SCALE;
+const STOP = ServiceActionItem.STOP;
```

Comment:
Why do we need to assign these to their own variables instead of just using `ServiceActionItem` directly?

Reply:
Thank you for taking the time to review. We don't need to, but I prefer defining them on top and not having `ServiceActionItem.SOMETHING` everywhere.